plugins/rhs_plugin/rhs_plugin.jsx

```diff
@@ -25,6 +28,7 @@ export default class RhsPlugin extends React.PureComponent {
                 className='sidebar-right__body'
             >
                 <SearchResultsHeader>
+                    {this.props.showAutoLinkedBoardTourTip ? <AutoShowLinkedBoardTourTip/> : null}
```

Comment:
@mhd-sln heads up this way I am adding the tourtip for the auto shown linked board